plugins/source/aws/resources/services/ses/contact_lists_fetch.go

```diff
@@ -13,27 +13,29 @@ func fetchSesContactLists(ctx context.Context, meta schema.ClientMeta, parent *s
 	c := meta.(*client.Client)
 	svc := c.Services().Sesv2
 
-	config := sesv2.ListContactListsInput{}
-	p := sesv2.NewListContactListsPaginator(svc, &config)
+	p := sesv2.NewListContactListsPaginator(svc, nil)
```

Comment:
Similar to other comments, this change seems to belong in another PR (assuming this one focuses on adding resources)